src/domain/challenge/space/space.resolver.mutations.ts

```diff
@@ -189,10 +191,24 @@ export class SpaceResolverMutations {
       AuthorizationPrivilege.UPDATE,
       `update spaceDefaults: ${space.id}`
     );
+    const account = await this.accountService.getAccountOrFail(
```

Comment:
Rather than reload the account entity + importing the account service, suggest to just expand the loading of the space 12 lines up to also have the defaults loaded in the relations?